app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,13 @@
+class NotificationsController < ApplicationController
+  def create
+    NotificationsMailer.contact(email_params).deliver_now
+
+    redirect_to  :back, notice: "Your email has been sent."
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.